main.tf

```diff
@@ -141,27 +141,33 @@ module "eks" {
   }
 
   ################################################################################
-  # Handle BYO IAM policy
+  # Handle BYO IAM Roles & Policies
   ################################################################################
-  create_iam_role                                = var.cluster_iam_role_name == null ? true : false   # v17: manage_cluster_iam_resources
-  iam_role_name                                  = var.cluster_iam_role_name                          # v17: cluster_iam_role_name
+  # BYO - EKS Cluster IAM Role
+  create_iam_role = var.cluster_iam_role_arn == null ? true : false # v17: manage_cluster_iam_resources
+  iam_role_arn    = var.cluster_iam_role_arn                        # v17: cluster_iam_role_arn
```

Comment:
For items with the v17 element can we remove these. They would have been useful during the upgrade. Thx.

Reply:
Removed additional comments for the cluster IAM role.